### PR TITLE
UIMARCAUTH-134: Define strict ESLint rules

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Add more ESLint rules that clearly define what code style we use use

## Issues
[UIMARCAUTH-134](https://issues.folio.org/browse/UIMARCAUTH-134)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -38,13 +41,33 @@
...
     "padding-line-between-statements": ["error",
       { "blankLine": "always", "prev": ["const", "let", "var"], "next": "*"},
       { "blankLine": "any", "prev": ["const", "let", "var"], "next": ["const", "let", "var"]}
-    ]
+    ],
+
+    "no-else-return": ["error", { "allowElseIf": false }],
+    "no-nested-ternary": "error",
+    "prefer-arrow-callback": "error",
+    "arrow-parens": [2, "always"],
```
Comment (Terala-Priyanka) [inline .eslintrc:54]: looks like this is conflicting with sonar cloud rule - 

**Braces and parentheses should be used consistently with arrow functions**,

defined at  rule https://sonarcloud.io/organizations/folio-org/rules?open=javascript%3AS3524&rule_key=javascript%3AS3524

Comment (BogdanDenis) [reply .eslintrc]: Thanks for noticing, changed the rule

Comment (Terala-Priyanka) [reply .eslintrc]: Welcome! There are more than 250 rules defined in sonar cloud - https://sonarcloud.io/organizations/folio-org/rules?languages=js
Should we have an equivalent eslint config file on each our repositories?

**Review by ODanyor — APPROVED** [review]

**Review by Svitlana-Korol — APPROVED** [review]

File: .eslintrc (hunk 1 above)
Comment (BogdanDenis) [reply .eslintrc]: @Terala-Priyanka I think so, I'll create additional tickets for our other modules to use the same rules

**Pull request merged**